contents/handbook/product/product-stages.md

```diff
@@ -0,0 +1,89 @@
+- Move fast, things might break, but fix issues quickly
+- Ruthlessly cut scope
+- Work with a small team (about 2 people)
+- Should move quickly based on gut/existing solutions in the market.
+
+When to go from an idea to an MVP:
+
+- TODO: add a link to the new section on which products we'll want to launch
+- Someone has conviction in the idea and envisions a rough path to PMF for our ICP
+- Take bets on ideas, understanding that some products may not reach PMF and could be deprecated
+
+### 2. Product-Market Fit (PMF)
+
+- Strong traction with Ideal Customer Profile (ICP) customers - we are commonly referenced in this category
+- Near feature parity with established tools
+- Differentiation mainly through integration with our other tools (If we can’t get to PMF through this then we consider doing bigger feature innovations)
+- Scalable, reliable, and profitable
+- API access for everything
+
+To progress from MVP to PMF:
+- Base decisions on customer feedback and market solutions
+- Prioritize features based on paying or potential paying customers' feedback
+- Focus on nailing the basics first
+- Begin charging once a user base is established
+- UX is familiar enough to existing solutions
+- Push hard, but recognize that some products may take time to reach PMF
+
+When should a product try to go from MVP to PMF:
+- Success for a product is reaching the PMF stage. We should only stop before that if the confidence in the product drops such as due to low uptake or high technical complexity
+
+### 3. Standalone Best-in-Class
+
+- Customers are signing up and paying for that specific tool
+- Frequently mentioned and rated as the best product in that category
+- On track for the largest market share among ICPs
+- Robust and reliable; major bugs rarely make it to production
+
+To advance from PMF to best-in-class:
+- Pay attention to small details
+- Innovate with major new feature bets
+- Conduct deep customer interviews to explore problems and find novel solutions
+
+When should a product try to go from PMF to standalone best-in-class:
+
+- Not every product must reach this stage; instead, focus on deep integration with other products for combined best-in-class status
+- Pursue standalone best-in-class when the product is meant to be a core growth engine, driving customer acquisition and word of mouth
+
+## Avoiding Overextension
+
+- What we don’t want to happen: a large number of products that never make it to PMF stage, are broken, unprofitable, poorly integrated, we don't use ourselves, and no one is driving it forward. If these are true we should use our judgment as to whether we should deprecate the feature.
+- Limit the number of MVPs and make it explicit to customers they are MVP stage so we don't detract from the quality of the rest of the product
+- The default goal of a new product should be to get it to PMF
+- Focus on startup ICPs, allowing quicker PMF with a reduced feature set
+- Be very selective about expanding our ICP
```

Comment:
Maybe "Be aggressive about deprecating products if we do not believe we'll get to PMF, if we don't have the capacity to support the product properly or if usage is very low"?